### PR TITLE
LPS-146406 Create a carousel component to quote comparison

### PR DESCRIPTION
**Pull request by silasbsi**

### CODE REVIEW

**Comment by silasbsi** [comment]
ci:test:sf

**Comment by silasbsi** [comment]
ci:test:sf

**Comment by silasbsi** [comment]
ci:test:sf

File: modules/apps/site-initializer/site-initializer-raylife/extra/remote-app/src/routes/quote-comparison/pages/QuoteComparison.js [hunk 1]
```diff
@@ -37,17 +50,33 @@ const QuoteComparison = () => {
 
 	const onClickPolicyDetails = () => {};
 
+	const cardElements = document.querySelectorAll('#quote-comparison');
```
Comment (kevenleone) [inline modules/apps/site-initializer/site-initializer-raylife/extra/remote-app/src/routes/quote-comparison/pages/QuoteComparison.js:53]:
```suggestion
	const quoteComparisonItems = document.querySelectorAll('#quote-comparison-item');
```

**Review by kevenleone — CHANGES_REQUESTED** [review]
@silasbsi see

**Comment by silasbsi** [comment]
ci:forward

**Pull request closed without merging**